### PR TITLE
Fix capturing references to stack variables

### PR DESCRIPTION
**Pull request by kheaactua**
Capture errors discovered by Aram's team.  I don't have Aram's (@akhzarj ?) contact info so I can't add a `Co-authored-by` tag in the commit

### CODE REVIEW

File: implementation/configuration/src/configuration_impl.cpp [hunk 1]
```diff
@@ -173,6 +174,8 @@ configuration_impl::configuration_impl(const configuration_impl &_other)
     sd_port_ = _other.sd_port_;
     sd_protocol_ = _other.sd_protocol_;
 
+    is_suppress_events_enabled_ = _other.is_suppress_events_enabled_;
+
```
Comment (duartenfonseca) [inline implementation/configuration/src/configuration_impl.cpp:178]: are this changes also part of the fix? as they were not mentioned on Aram's comment https://github.com/COVESA/vsomeip/issues/730

Comment (kheaactua) [reply implementation/configuration/src/configuration_impl.cpp:178]: They were discussed in the vsomeip meeting at the same time he was discussing the other issues.

**Comment by akhzarj** [comment]
Hi @kheaactua 
Please find our contacts related to Co-Authority:
- Aram Khzarjyan <aram.khzarjyan@mercedes-benz.com>
- Patnaik, Sambit  <sambit.patnaik@mercedes-benz.com>

**Comment by kheaactua** [comment]
> Hi @kheaactua Please find our contacts related to Co-Authority:
> 
>     * Aram Khzarjyan [aram.khzarjyan@mercedes-benz.com](mailto:aram.khzarjyan@mercedes-benz.com)
> 
>     * Patnaik, Sambit  [sambit.patnaik@mercedes-benz.com](mailto:sambit.patnaik@mercedes-benz.com)

Got it!  Thanks!

image

**Pull request merged**